src/Traits/HasAdditionalTransactionData.php

```diff
@@ -69,4 +71,28 @@ public function addAdditionalTransactionData($name, $value): self
         $this->additionalTransactionData->$name = $value;
         return $this;
     }
+
+    public function setShipping(?ShippingData $shipping): self
```

Comment:
Please provide function headers, they should be present automatically if you use the proper template: e.g. "pubf", "prif", etc.